modules/builds-create-custom-build-artifacts.adoc

```diff
@@ -14,7 +14,7 @@ You must create the image you want to use as your custom build image.
 +
 [source,terminal]
 ----
-FROM registry.centos.org/centos:7
+FROM registry.redhat.io/rhel8/buildah
 RUN yum install -y buildah
```

Comment:
We don't need `RUN yum install -y buildah` - the image already contains buildah and its dependencies.

Reply:
@adambkaplan I removed the line. In addition in line 34 I noticed the path is is written `/tmp/built`. Assuming that's a typo I changed that to `/tmp/build`.

Reply:
@adambkaplan Can you QE review? 
I need that for the peer-review by the docs team.